src/main/java/book/project/bookstore/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package book.project.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use `@Data` for entities
```suggestion
@Getter
@Setter
@Table(name...
```

Reply:
fix comment